content/en/docs/reference/resource-types/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Resource Types
+---
+
+<!-- overview -->
+
+This section provides information on the different resource types you can request, built into Kubernetes.
```

Comment:
```suggestion
This page provides information on the different resource types you can request, built into Kubernetes.
```

Reply:
Maybe this page should also mention that [extended resources](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#extended-resources) exist?

Reply:
I agree!